docs/clients.md

```diff
@@ -12,4 +12,5 @@
 
 In this section you can find short code samples to interact with OpenVINO Model Server endpoints via:
 - [TensorFlow Serving API](./clients_tfs.md)
-- [KServe API](./clients_kfs.md)
\ No newline at end of file
+- [KServe API](./clients_kfs.md)
+git 
```

Comment:
Is it valid change or was it made by accident?

Reply:
just fixed